client/src/containers/add-building/FormFragment.js

```diff
@@ -67,6 +68,9 @@ const FormFragment = ({ form }) => {
     setCoordinates(newCoordinates);
   };
 
+  // eslint-disable-next-line no-unused-vars
+  const handleVerifyCaptcha = (token, eKey) => {};
```

Comment:
Don't we have to do something with the token ?

Reply:
I think the method should somehow check that once the captcha was verified the form can be submitted.
Right now it seems that, [in the preview, submitting a new building](https://seismic-risc-rlamg8au2-code4romania.vercel.app/adauga-cladire) isn't allowed even after completing the captcha 

![Screenshot 2021-06-29 at 18 25 16](https://user-images.githubusercontent.com/2820084/123825185-5e002b80-d907-11eb-8ae0-3be3205fc509.png)